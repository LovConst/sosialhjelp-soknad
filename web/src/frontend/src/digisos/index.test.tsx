import * as React from "react";
import * as ReactDOM from "react-dom";
import IntlProvider from "../intlProvider";

it("renders without crashing", () => {
	const div = document.createElement("div");
	ReactDOM.render(
		<IntlProvider>
			<div />
		</IntlProvider>,
		div
	);
});
