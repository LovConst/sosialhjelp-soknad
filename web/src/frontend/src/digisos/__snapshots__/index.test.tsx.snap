// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Connect(IntlProvider)>
    <InjectIntl(App)
      intl={null}
    />
  </Connect(IntlProvider)>,
  "nodes": Array [
    <Connect(IntlProvider)>
      <InjectIntl(App)
        intl={null}
      />
    </Connect(IntlProvider)>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Provider
        store={
          Object {
            "dispatch": [Function],
            "getState": [Function],
            "replaceReducer": [Function],
            "subscribe": [Function],
            Symbol(observable): [Function],
          }
        }
      >
        <Connect(IntlProvider)>
          <InjectIntl(App)
            intl={null}
          />
        </Connect(IntlProvider)>
      </Provider>,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Provider {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "children": <Connect(IntlProvider)>
            <InjectIntl(App)
              intl={null}
            />
          </Connect(IntlProvider)>,
          "store": Object {
            "dispatch": [Function],
            "getState": [Function],
            "replaceReducer": [Function],
            "subscribe": [Function],
            Symbol(observable): [Function],
          },
        },
        "refs": Object {},
        "state": null,
        "store": Object {
          "dispatch": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
          Symbol(observable): [Function],
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Connect(IntlProvider)>
          <InjectIntl(App)
            intl={null}
          />
        </Connect(IntlProvider)>,
        "_debugID": 2,
        "_renderedOutput": <Connect(IntlProvider)>
          <InjectIntl(App)
            intl={null}
          />
        </Connect(IntlProvider)>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Provider
    store={
      Object {
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
        Symbol(observable): [Function],
      }
    }
  >
    <Connect(IntlProvider)>
      <InjectIntl(App)
        intl={null}
      />
    </Connect(IntlProvider)>
  </Provider>,
}
`;
